resources/zib/zib-wounds.WoundImage.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-wounds.WoundImage" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-wounds.WoundImage" />
+  <name value="ZibwoundsWoundImage" />
+  <title value="zib wounds.WoundImage" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A photo of the wound, burn wound or decubitus wound as visual information.&#xD;&#xA;&#xD;&#xA;* [Burnwound v3.4(2020EN)](https://zibs.nl/wiki/Burnwound-v3.4(2020EN))&#xD;&#xA;* [PressureUlcer v3.4(2020EN)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN))&#xD;&#xA;* [Wound v3.3(2020EN)](https://zibs.nl/wiki/Wound-v3.3(2020EN))" />
+  <purpose value="This DocumentReference resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Burnwound v3.4 (2020)](https://zibs.nl/wiki/Burnwound-v3.4(2020EN)), PressureUlcer v3.4 (2020)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN)), and Wound v3.3 (2020)](https://zibs.nl/wiki/Wound-v3.3(2020EN))." />

```

Comment:
This resource does not represent the entire zibs, but only the WoundImage concept within those zibs. Please use a similar text as in zib-LaboratoryTestResult.Specimen. In particular make sure that the brackets are used correctly.

Reply:
Fix